libs/vc-helper/src/main/java/uk/gov/di/ipv/core/library/vchelper/VcHelper.java

```diff
@@ -37,9 +37,17 @@ public static void setConfigService(ConfigService configService) {
         VcHelper.configService = configService;
     }
 
-    private static Set<String> getNonEvidenceCredentialIssuers() {
+    public static Set<String> getNonEvidenceCredentialIssuers() {
+        return VcHelper.getNonEvidenceCredentialIssuers(configService);
+    }
+
+    public static Set<String> getNonEvidenceCredentialIssuers(ConfigService configService) {
         return NON_EVIDENCE_CRI_TYPES.stream()
-                .map(credentialIssuer -> configService.getComponentId(credentialIssuer))
+                .map(
+                        credentialIssuer ->
+                                configService
+                                        .getCredentialIssuerActiveConnectionConfig(credentialIssuer)
+                                        .getComponentId())
```

Comment:
I think this should stay as a call to `configService.getComponentId()`, rather than getting the active connection then getting the component ID. Functionally the same but less verbose and it might dry up a lot of the mocking you've had to do in the tests.

Reply:
Done.